code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -225,17 +225,20 @@
 
 /obj/item/clothing/head/helmet/space/plasmaman/atmospherics
 	name = "atmospherics plasma envirosuit helmet"
-	desc = "A space-worthy helmet specially designed for atmos technician plasmamen, the usual purple stripes being replaced by atmos' blue."
+	desc = "A space-worthy helmet specially designed for atmos technician plasmamen, the usual purple stripes being replaced by atmos' blue. Has improved thermal shielding." // Added the thermal shielding clarification
```

Comment:
Don't make comments that describe what your pr does, they're for people in the future, not now.